### PR TITLE
Ackee audit fixes by @vgorkavenko

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: test/CSBondCurve.t.sol [hunk 1]
```diff
@@ -66,6 +66,11 @@ contract CSBondCurveTest is Test {
         bondCurve.initialize(_bondCurve);
     }
 
+    function test_getCurveInfo_RevertWhen_InvalidBondCurveId() public {
+        vm.expectRevert(CSBondCurve.InvalidBondCurveId.selector);
+        bondCurve.getCurveInfo(1337);
+    }
+
```
Comment (dgusakov) [inline test/CSBondCurve.t.sol:73]: Add a positive test also. I mean getting the existing curve. I understand that this is implicitly tested in the other tests, but why not have it here explicitly?

File: src/lib/base-oracle/HashConsensus.sol [hunk 2]
```diff
@@ -87,6 +24,7 @@ interface IReportAsyncProcessor {
 /// given that oracle reports sometimes have to contain diffs instead of the full state which
 /// might be impractical or even impossible to transmit and process.
 ///
+// solhint-disable ordering
```
Comment (dgusakov) [inline src/lib/base-oracle/HashConsensus.sol:27]: Should we also fix "I8: HashConsensus should inherit from IConsensusContract" here?

File: src/CSModule.sol [hunk 3]
```diff
@@ -1047,20 +1047,21 @@ contract CSModule is
     function settleELRewardsStealingPenalty(
         uint256[] calldata nodeOperatorIds
     ) external onlyRole(SETTLE_EL_REWARDS_STEALING_PENALTY_ROLE) {
+        ICSAccounting _accounting = accounting;
```
Comment (dgusakov) [inline src/CSModule.sol:1050]: How much does this move save in terms of gas? Just curious

Comment (vgorkavenko) [reply src/CSModule.sol:1050]: ### Before

| src/CSModule.sol:CSModule contract      |                 |        |        |        |         |
|-----------------------------------------|-----------------|--------|--------|--------|---------|
| Deployment Cost                         | Deployment Size |        |        |        |         |
| 5285863                                 | 24987           |        |        |        |         |
| Function Name                           | min             | avg    | median | max    | # calls |
| settleELRewardsStealingPenalty          | 24494           | 82423  | 70685  | 150775 | 13      |

### After

| src/CSModule.sol:CSModule contract      |                 |        |        |        |         |
|-----------------------------------------|-----------------|--------|--------|--------|---------|
| Deployment Cost                         | Deployment Size |        |        |        |         |
| 5286739                                 | 24991           |        |        |        |         |
| Function Name                           | min             | avg    | median | max    | # calls |
| settleELRewardsStealingPenalty          | 24494           | 82395  | 70503  | 150287 | 13      |

File: src/lib/base-oracle/HashConsensus.sol (hunk 2 above)
Comment (vgorkavenko) [reply src/lib/base-oracle/HashConsensus.sol:28]: added

Comment (dgusakov) [reply src/lib/base-oracle/HashConsensus.sol:28]: is it added?

**Review by dgusakov — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**